src/Core/tests/DeviceTests/Handlers/DatePicker/DatePickerHandlerTests.Windows.cs

```diff
@@ -12,7 +12,7 @@ public partial class DatePickerHandlerTests
 		[Theory(DisplayName = "Format Initializes Correctly")]
 		[InlineData("dd/MM/yyyy", "{day.integer(2)}/{month.integer(2)}/{year.full}")]
 		[InlineData("d/M/yy", "{day.integer}/{month.integer(1)}/{year.abbreviated}")]
-		[InlineData("ddd/MMM/yyyy", "{day.integer} {dayofweek.abbreviated}/{month.abbreviated}/{year.full}")]
+		[InlineData("ddd/MMM/yyyy", "{dayofweek.abbreviated}/{month.abbreviated}/{year.full}")]
```

Comment:
The test previously passed with
`[InlineData("ddd/MMM/yyyy", "{day.integer} {dayofweek.abbreviated}/{month.abbreviated}/{year.full}")]`
?

Should we validate the resulting string as well or include a different test?

Reply:
This test validates that the input string gets translated to the expected output, which should be what I changed it to here (hence why I did). So to validate the existing string, it would be a new test to validate that the previous string that was there (`{day.integer} {dayofweek.abbreviated}`) is incorrect, so we don't regress it when making changes.

I'm not sure how important that is, but I'm not against doing it.